src/applications/personalization/profile-2/tests/components/DirectDepositContent.unit.spec.jsx

```diff
@@ -1,98 +1,242 @@
 
+import {
+  mockFetch,
+  resetFetch,
+  setFetchJSONFailure,
+  setFetchJSONResponse,
+} from 'platform/testing/unit/helpers';
+import { commonReducer } from 'platform/startup/store';
+
+import reducer from 'applications/personalization/profile360/reducers';
+
+import profileUi from '../../reducers';
 import DirectDepositContent from '../../components/direct-deposit/DirectDepositContent';
 
-describe('DirectDepositContent', () => {
-  let wrapper;
-  let store;
-  let state;
-  const makeFakeStore = () => {
-    state = {
-      user: {
-        login: {
-          currentlyLoggedIn: true,
+function createBasicInitialState() {
+  return {
+    user: {
+      profile: {
+        loa: {
+          current: 3,
+          highest: 3,
         },
-        profile: {
-          loa: {
-            current: 3,
-            highest: 3,
+        multifactor: true,
+      },
+    },
+    vaProfile: {
+      paymentInformation: {
+        responses: [
+          {
+            paymentAccount: {
+              accountType: 'Checking',
+              financialInstitutionName: 'Bank of EVSS',
+              accountNumber: '****5678',
+              financialInstitutionRoutingNumber: '*****0021',
+            },
           },
-          multifactor: true,
-        },
+        ],
+      },
+      paymentInformationUiState: {},
+    },
+  };
+}
+
+function setUpWithInitialState(initialState = createBasicInitialState()) {
+  const store = createStore(
+    combineReducers({ ...commonReducer, ...reducer, profileUi }),
+    initialState,
+    applyMiddleware(thunk),
+  );
+  return mount(
+    <Provider store={store}>
+      <DirectDepositContent />
+    </Provider>,
+  );
+}
+
+function fillOutAndSubmitBankInfoForm(form) {
+  const routingNumberInput = form.find('input#root_routingNumber');
+  const accountNumberInput = form.find('input#root_accountNumber');
+  const accountTypeSelect = form.find('select#root_accountType');
+
+  // enter valid data in the bank info form...
+  routingNumberInput.simulate('change', {
+    target: { name: 'root_routingNumber', value: '123123123' },
+  });
+  accountNumberInput.simulate('change', {
+    target: { name: 'root_accountNumber', value: '123123123' },
+  });
+  accountTypeSelect.simulate('change', {
+    target: { name: 'root_accountType', value: 'Savings' },
+  });
+
+  form.simulate('submit');
+}
+
+// Thanks Pivotal http://engineering.pivotal.io/post/react-integration-tests-with-enzyme/
+function asyncFlush() {
+  return new Promise(resolve => setTimeout(resolve, 0));
+}
+
+describe('DirectDepositContent', () => {
+  let initialState;
+  it('should render nothing if the user is LOA1', () => {
+    initialState = createBasicInitialState();
+    initialState.user.profile = {
+      loa: {
+        current: 1,
+        highest: 3,
       },
-      vaProfile: {
-        paymentInformation: {
+    };
+
+    const wrapper = setUpWithInitialState(initialState);
+
+    expect(wrapper.html()).to.equal('');
```

Comment:
This works, but I have used `isEmptyRender` in the past https://enzymejs.github.io/enzyme/docs/api/ReactWrapper/isEmptyRender.html

Reply:
Thanks, that is better. Updated!